oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -165,6 +165,22 @@ public interface CentralProcessor {
      */
     double[] getSystemLoadAverage(int nelem);
 
+    /**
+     * This method waits the provided amount of milliseconds
+     * and then returns the cpu usage for that time period.
+     *
+     * @param delay
+     *          Milliseconds to wait.
+     * @return Value between 0 and 1 (100%) that represents the cpu usage in the provided time period.
+     * @see #getSystemCpuLoadTicks()
+     * @see #getSystemCpuLoadBetweenTicks(long[])
+     */
+    default double getSystemCpuLoad(long delay) {
+        long[] oldTicks = getSystemCpuLoadTicks();
+        Util.sleep(delay);
```

Comment:
A suggestion here.   The `getSystemCpuLoadTicks()` call takes a few milliseconds so the overall time period is going to be slightly more than the specified delay.   For example, if it takes 25ms to fetch the ticks, timestamped somewhere near the end of that period, you might get the following sequence:
 - Initial get call. 20ms later, ticks collected
 - 5 ms later, method returns
 - 1000ms delay
 - Final get call.  20ms later, ticks collected
 - 5 ms later, method returns

Total ticks collected end up being 1025 (5ms at tail end of initial call, 20ms at beginning of final call).

This can be corrected using the following sequence:
```java
long start = System.nanoTime();
long[] oldTicks = getSystemCpuLoadTicks();
long toWait = (System.nanoTime() - start) / 1_000_000;
// protect against IllegalArgumentException
if (toWait > 0L) {
    Util.sleep(toWait);
}
```